multimedia/views.py

```diff
@@ -4,6 +4,8 @@
 )
 from crew.models import Crew
 
+no_preview_150 = '/statics/images/no-preview-150.png'
+no_preview = '/statics/images/no-preview.png'
```

Comment:
Can try `{% static \"images/no-preview.png\" %}` here for me? I think this is the better way, if possible